lib/sport_ngin_aws_auditor/instance.rb

```diff
@@ -14,8 +14,9 @@ def initialize(type, data_array)
         self.name = data_array[1] || nil
         self.reason = data_array[2] || nil
         self.tag_value = data_array[3] || nil
+        self.region_based = false
       else
-        self.type = type
+        self.region_based = data_array[1] || nil
```

Comment:
What is `data_array` and where does that come from?  Above `[1]` is the name but in this case its a boolean(?).  That array smells of over complexity which makes it hard to read and is at risk for bugs.